django_tasks/backends/base.py

```diff
@@ -27,6 +27,9 @@ class BaseTaskBackend(metaclass=ABCMeta):
     supports_get_result = False
     """Can results be retrieved after the fact (from **any** thread / process)"""
 
+    supports_signed_task = False
```

Comment:
Suggestion: Because signing is strictly an internal operation, I don't think this needs to be something a backend "supports".